packages/components/psammead-inline-link/src/index.stories.jsx

```diff
@@ -1,12 +1,43 @@
 import React from 'react';
+import styled from 'styled-components';
 import { storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import { getBodyCopy, getLongPrimer } from '@bbc/gel-foundations/typography';
+import {
+  getSansRegular,
+  getSansBold,
+  getSansRegularItalic,
+  getSansBoldItalic,
+} from '@bbc/psammead-styles/font-styles';
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
 import notes from '../README.md';
 import InlineLink from './index';
 
+const RegularParagraph = styled.p`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansRegular(service)}
+`;
+const BoldParagraph = styled.p`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansBold(service)}
+`;
+const ItalicParagraph = styled.p`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansRegularItalic(service)}
+`;
+const BoldItalicParagraph = styled.p`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansBoldItalic(service)}
+`;
+
+const Caption = styled.p`
```

Comment:
Since you've called this a caption, it'd be good to have the element be one, too.
```suggestion
const Caption = styled.figcaption`
```
Everything else looks great. 👍 Reviewed this with UX and they agree that it's very helpful for them to review the different styling implementations.